db/migrate/20220704130218_add_number_of_available_points_to_employees.rb

```diff
@@ -0,0 +1,5 @@
+class AddNumberOfAvailablePointsToEmployees < ActiveRecord::Migration[6.1]
+  def change
+    add_column :employees, :number_of_available_points, :decimal, null: false, default: 0, precision: 8, scale: 2
```

Comment:
I would simply name `:available_points` , also not sure if we really need decimal here - probably int would be enough
```suggestion
    add_column :employees, :available_points, :decimal, null: false, default: 0, precision: 8, scale: 2
```

Reply:
Because in the story employee can buy a reward with points I decided to treat points as currency and use decimals in the database. Thanks to that employees can purchase a reward that can cost less than one point. Because I decided to make an extra field in the table I can save the number of available points independent of the kudos number received.   When an employee buys rewards I can reduce the number of available points and save them in the database.